plugins/spark/v3.5/spark/build.gradle.kts

```diff
@@ -46,6 +46,7 @@ dependencies {
   // TODO: extract a polaris-rest module as a thin layer for
   //  client to depends on.
   implementation(project(":polaris-core")) { isTransitive = false }
+  testImplementation("org.apache.hudi:hudi-spark3.5-bundle_${scalaVersion}:1.1.0")
```

Comment:
Nit: we put versions in the file `pluginlibs.versions.toml`, refer it as line 35 does.